README.md

```diff
@@ -11,6 +11,9 @@ Gradle Tasks
 `./gradlew nativeImage`: create a native image using GraalVM's `native-image` tool with the configuration as specified
 by the `graal` Gradle extension. Outputs are produced to `${projectDir}/build/graal/`.
 
+`./gradlew sharedLibary`: * `sharedLibrary`: runs GraalVM's `native-image` for `shared library` command with configured 
```

Comment:
this sentence doesn't quite parse for me. how about:

like the `nativeImage` task, but configures the `native-image` tool to produce a shared library

Reply:
Can you help me with a better sentence. My english is not very good :(

Reply:
of course. you can just use the one I wrote above:

like the `nativeImage` task, but configures the `native-image` tool to produce a shared library